profiles/api/serializers/billing_group_serializers.py

```diff
@@ -6,4 +6,16 @@
 class BillingGroupSerializer(serializers.ModelSerializer):
     class Meta:
         model = BillingGroup
-        fields = '__all__'
+        fields = ['id', 'name', 'user_set', 'quota_bindings']
+
+
+class BillingGroupWriteSerializer(serializers.ModelSerializer):
```

Comment:
Switch to read serializer (full nested object) and the default serializer is the one used for object creation